Assignment2/Assignment2.yaml

```diff
@@ -0,0 +1,235 @@
+swagger: "2.0"
+info:
+  description: "Welcome to the swapi, the Star Wars API! This documentation should help you familiarise yourself with the resources available and how to consume them with HTTP requests. If you're after a native helper library then I suggest you scroll down and check out what's available. Read through the getting started section before you dive in. Most of your problems should be solved just by reading through it.."    
+  version: "1.0.0"
+  title: "swapi"
+  termsOfService: "http://swagger.io/terms/"
+  contact:
+    email: "apiteam@swagger.io"
+  license:
+    name: "Apache 2.0"
+    url: "http://www.apache.org/licenses/LICENSE-2.0.html"
+host: "swapi.co"
+basePath: "/api"
+tags: 
+- name: "people"
+  description: "Create Documentation of swapi APIs"
+  externalDocs:
+    description: "Find out more"
+    url: "http://swapi.co"
+- name: "films"
+  description: "Access to Star Wars film resources"
+- name: "species"
+  description: "Information about species"
+  externalDocs:
+    description: "Type of person in the Star War Universe"
+    url: "http://swapi.co"
+schemes:
+- "http"
+paths:
+  /people:
+    
+    get:
+      tags:
+      - "people"
+      summary: "Fetches all people"
+      description: "Returns all people resources of the Star War apis"
+      operationId: "getPeople"
+      produces:
+      - "application/xml"
+      - "application/json"
+      
+      responses:
+        200:
+          description: "successful operation"
+          schema:
+            $ref: "#/definitions/People"
```

Comment:
Incorrect Schema attached in response! This APIs returns Array of People, not just People.